instrumentation/servlet/servlet-3.0/javaagent-unit-tests/src/test/java/io/opentelemetry/javaagent/instrumentation/servlet/v3_0/snippet/TestUtil.java

```diff
@@ -0,0 +1,21 @@
+package io.opentelemetry.javaagent.instrumentation.servlet.v3_0.snippet;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class TestUtil {
+  public static String readFile(String resourceName) throws IOException {
+    InputStream is =
```

Comment:
(more conventional short name)
```suggestion
    InputStream in =
```